doc/developer-guides/hld/hld-security.rst

```diff
@@ -274,16 +278,14 @@ following rules:
 #. Verify that the Service VM is a closed system and doesn't allow the user to
    install any unauthorized third-party software or components.
 #. Verify that external peripherals are constrained.
-#. Enable kernel-based hardening techniques, for example dm-verity (to
-   ensure integrity of the DM and vBIOS/vOSloaders), and kernel module
+#. Enable kernel-based hardening techniques, for example, dm-verity (to
+   ensure the integrity of the DM and vBIOS/vOSloaders), and kernel module
    signing.
 #. Enable system level hardening such as MAC (Mandatory Access Control).
 
 Detailed configurations and policies are out of scope for this document.
 For good references on OS system security hardening and enhancement,
-see `AGL security
-<https://docs.automotivelinux.org/docs/en/master/architecture/reference/security/part-2/0_Abstract.html>`_
-and `Android security <https://source.android.com/security/>`_.
+see `Android security <https://source.android.com/security/>`_.
 
```

Comment:
@dbkinder I removed the AGL link because it's broken. Is there a replacement link?

Reply:
Looks like the AGL security documentation is here:  https://docs.automotivelinux.org/en/lamprey/#2_Architecture_Guides/2_Security_Blueprint/0_Overview/